### DISCUSSION TITLE
Payout not schedule

### DISCUSSION THREAD

**Discussion by serenader2014** [community/community#113956] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have created my sponsor account last year, and connected a stripe account, but in the payout account, I've entered the wrong account number, so I still not receiving the money. Today I finally figure out how to add the correct account number, but the stripe is not schedule a new payout, how can I know when will stripe issue a new payout to my account?

**Comment by anjuan** [comment]
Hi @serenader2014, you may want to contact Stripe Support. They can help you understand how they handled the funds that were sent to the wrong account number.
